stl/inc/system_error

```diff
@@ -556,7 +555,7 @@ _NODISCARD const _Ty& _Immortalize_memcpy_image() noexcept {
     static constexpr _Ty _Static;
     return _Static;
 }
-#elif defined(_MSC_VER) && _MSC_VER > 1927 && !defined(_M_CEE) // _M_CEE test is TRANSITION, VSO-1153256
+#elif !defined(_M_CEE) && !defined(__clang__) // TRANSITION, VSO-1153256 and LLVM issue not yet filed as of 2020-08-06

```

Comment:
CUDA is ok with this branch?

Reply:
We'll see when I port this to the MSVC-internal repo where our CUDA test coverage currently runs. I *believe* it will work - the CUDA compilation model uses EDG to split the source code for compilation by the device and host compilers. Unrecognized compiler builtins are a problem for this, but attributes are Standard syntax, so they should survive the splitting process.

Reply:
x86 and x64 internal test runs of `stl`, `vcr`, and `compiler` passed, so CUDA appears to be happy.